lib/Agrammon/ModuleBuilder.pm6

```diff
@@ -12,8 +12,10 @@ class Agrammon::ModuleBuilder {
         input => { :enum }
     }
 
+    my Int $module-load-order = 0;
```

Comment:
Hm, this is global and not going to be safe if we were to, for example, load to models concurrently (maybe not a big concern right now, but better to do things right than have a subtle bug in the future). Probably should become an instance field and we can `.new` the `ModuleBuilder` actions class in some appropriate place (perhaps holding one per `Agrammon::Model` instance and reusing it).